pkg/updater/resultstore/resultstore_test.go

```diff
@@ -488,6 +488,119 @@ func TestResultStoreColumnReader(t *testing.T) {
 	}
 }
 
+func TestCellMessageIcon(t *testing.T) {
+	cases := []struct {
+		name        string
+		annotations []*configpb.TestGroup_TestAnnotation
+		properties  map[string][]string
+		tags        []string
+		message     string
+		icon        string
+	}{
+		{
+			name: "basically works",
+		},
+		{
+			name: "find annotation from property",
+			annotations: []*configpb.TestGroup_TestAnnotation{
+				{
+					ShortText: "icon",
+					ShortTextMessageSource: &configpb.TestGroup_TestAnnotation_PropertyName{
+						PropertyName: "props",
+					},
+				},
+			},
+			properties: map[string][]string{
+				"props": {"first", "second"},
+			},
+			message: "first",
+			icon:    "icon",
+		},
+		{
+			name: "find annotation from tag",
+			annotations: []*configpb.TestGroup_TestAnnotation{
+				{
+					ShortText: "icon",
+					ShortTextMessageSource: &configpb.TestGroup_TestAnnotation_PropertyName{
+						PropertyName: "actually-a-tag",
+					},
+				},
+			},
+			tags:    []string{"actually-a-tag"},
+			message: "actually-a-tag",
+			icon:    "icon",
+		},
+		{
+			name: "find from pass_with_errors",
```

Comment:
These and the other test cases that reference well-known properties can also be removed.